packages/block-library/src/cover/edit.js

```diff
@@ -329,32 +393,26 @@ class CoverEdit extends Component {
 		return (
 			<>
 				{ controls }
-				<ResizableBox
+				<ResizableCover
 					className={ classnames(
 						'block-library-cover__resize-container',
 						{ 'is-selected': isSelected },
 					) }
-					size={ {
-						height: minHeight,
```

Comment:
Wouldn't it make sense to just delete this to fix the issue?

Reply:
I tried that and many other ways to fix the problem and this was the only solution that worked well. It is not straightforward to use re-resizable with min-heights instead of heights.